### PR TITLE
Library v1

### PR DESCRIPTION
**Pull request by tmampa**
## Added HTML, CSS, and script files
 # THE APP CAN
- Add new books
- View all books
- Remove books
- Change read status

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
+/* eslint-disable func-names */
+/* eslint-disable no-nested-ternary */
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-plusplus */
```
Comment (afanimartin) [inline js/app.js]: - Fix all these linter warnings. Disabling the rules is not a good practice.

File: js/app.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+const inputTitle = document.getElementById('title');
+const inputAuthor = document.getElementById('author');
+const inputPages = document.getElementById('pages');
+const inputReadedYes = document.getElementById('readedYes');
+const inputReadedNo = document.getElementById('readedNo');
+const buttonSubmit = document.getElementById('buttonSubmitBook');
+const warning = document.getElementById('warning');
+const buttonAddBook = document.getElementById('buttonAddBook');
+
+/* *********** LOCAL STORAGE *********** */
```
Comment (afanimartin) [inline js/app.js]: Remove all comments in the codebase. Comments make our code look ugly

**Review by afanimartin — CHANGES_REQUESTED** [review]
## changes required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this PR. Let's work on the following changes before we can merge.

### changes 🔕 🔕 🔕 
- books created don't show in the UI. Make sure your fix the bug with rendering books created
- you have a spelling error: `Is readed?`. Change it to `Is read?`. This is in the form for creating a new book


### Martin Afani
#### Slack: whiz25

File: js/index.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+function clearInputs() {
+  inputTitle.value = '';
+  inputAuthor.value = '';
+  inputPages.value = '';
+  warning.textContent = '';
+  if (inputreadYes.checked) inputreadYes.checked = false;
+  else inputreadNo.checked = false;
+}
+
+function displayBook(book) {
```
Comment (rukundoeric) [inline js/index.js:76]: - [ ] <b>OPTIONAL:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```

**Review by rukundoeric — APPROVED** [review]
Hi @tmampa,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] check the comments under review


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**